static/js/factories/ecommerce.js

```diff
@@ -173,7 +173,8 @@ export const makeCouponPaymentVersion = (
   payment_transaction: casual.word,
   company:             casual.word,
   created_on:          casual.moment.format(),
-  updated_on:          casual.moment.format()
+  updated_on:          casual.moment.format(),
+  include_future_runs: true
```

Comment:
@pdpinch should 'Enable future runs' be checked (True) by default on the coupon creation form?

Reply:
No. I think I should actively check it when we need it.

Reply:
done